src/store/wizardSlice.ts

```diff
@@ -67,7 +71,10 @@ type wizardState = {
       enabled: string[] | undefined;
     };
   };
-
+  fileSystem: {
+    mode: FileSystemPartitionMode;
+    fileSystemConfiguration: FileSystemConfigInfo[];
```

Comment:
This will be very nit picky, but I'm thinking about more descriptive names. What do you think about 
```typescript
fsc: {
  partitioningMode: 'automatic',
  fileSystemConfiguration: [
    ...
  ],
}
```

for the step name from what I can see we are mirroring the headers on the step mostly and mode -> partitioningMode is a small change, but makes things clearer even without any context.

Reply:
amm @lucasgarfield advice me to change it to mode, @lucasgarfield WDYT?